rpcd/playbooks/ironic-pre-config.yml

```diff
@@ -0,0 +1,125 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+- name: Pre install config for ironic
+  hosts: ironic_all
```

Comment:
If we're doing pre installation tasks, this should be done on localhost (the deployment node) instead of the ironic hosts.